packages/client/src/workflow-client.ts

```diff
@@ -275,7 +309,14 @@ export class WorkflowClient {
     // Cast is needed because it's impossible to deduce the type in this situation
     const interceptors = (this.options.interceptors.calls ?? []).map((ctor) => ctor({ workflowId }));
     const runId = await this._start(workflowTypeOrFunc, { ...options, workflowId }, interceptors);
-    const handle = this._createWorkflowHandle(workflowId, runId, interceptors, {
+    // runId is not used in handles created with `start*` calls because these
+    // handles should be allow interacting with the workflow if it continues as new.
+    const handle = this._createWorkflowHandle({
+      workflowId,
+      runId: undefined,
```

Comment:
I wonder if we want to document that if you want to cancel/terminate/describe/etc a specific run ID, you must use `getHandle`.

Reply:
how else would you do it with public api's?

Reply:
Can you clarify the question?

What I mean is previously if you did a start + describe, you'd be describing the run that resulted from the start (even if it had continue-as-new'd). Now you'll be describing the latest run for the workflow ID. That seems like the right behavior, but I just wonder if we need to make it clear that you're not necessarily describing the result of start, you're describing the latest.

Reply:
ah ok. i misread your first comment then.